ambry-clustermap/src/main/java/com.github.ambry.clustermap/StaticClusterManager.java

```diff
@@ -101,6 +101,14 @@ public byte getLocalDatacenterId() {
     return localDatacenterId;
   }
 
+  @Override
+  public String getDatacenterName(byte id) {
+    if (id == ClusterMapUtils.UNKNOWN_DATACENTER_ID) {
+      return null;
+    }
+    return hardwareLayout.getDatacenterById(id).getName();
```

Comment:
this could throw a null pointer exception if a datacenter with that id was not found

Reply:
Will use if else to detect null.